dex-test-common/src/main/scala/com/wavesplatform/dex/test/matchers/DiffMatcherWithImplicits.scala

```diff
@@ -7,9 +7,8 @@ import org.scalatest.matchers.{MatchResult, Matcher}
 
 trait DiffMatcherWithImplicits {
 
-  private val byteStrDiff: Diff[ByteStr] = getDiff[ByteStr](_.toString == _.toString)
-
-  implicit val derivedByteStrDiff: Derived[Diff[ByteStr]] = Derived(byteStrDiff)
+  implicit val derivedByteStrDiff: Derived[Diff[ByteStr]] = Derived(getDiff[ByteStr](_.toString == _.toString))
+  implicit val derivedStringDiff: Derived[Diff[String]]   = Derived(getDiff[String](_ == _))
```

Comment:
It's really needed? :)

Reply:
Otherwise WebSocketMessagesSerdeSpecification (_string_ status in WsOrder) and MathcerSettingsSpecification (_string_ target in GrpcClientSettings) are broken in misterious way

We haven't seen error messages because they were somehow suppressed, they were just marked as ignored (try it out in master branch!)

Reply:
Ok

Reply:
Could you add a comment and TODO ?

Reply:
Done